### DISCUSSION TITLE
raise TypeError(TypeError: Incompatible collection type: str is not list-like  when creating a post using sql db engine

### DISCUSSION THREAD

**Discussion by Timmy-Edibo** [fastapi/fastapi#8536] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import Depends, FastAPI, HTTPException, status
from sqlalchemy.orm import Session

from . import crud, models, schemas
from .database import SessionLocal, engine

from typing import List
from sqlalchemy.orm import joinedload

models.Base.metadata.create_all(bind=engine)



app = FastAPI()


# Dependency
def get_db():
    db = SessionLocal()
    try:
        yield db
    finally:
        db.close()


@app.post('/author/')
async def create_author(form: schemas.Author, db: Session = Depends(get_db)):
    create_author = models.Author(**form.dict())

    db.add(create_author)
    db.commit()
    db.refresh(create_author)
    return create_author
    
    
@app.post('/books')
async def create_book(form: schemas.Book, db: Session = Depends(get_db)):
    create_book = models.Book(**form.dict())

    db.add(create_book)
    db.commit()
    db.refresh(create_book)
    return create_book


@app.get("/books/{id}", response_model=schemas.BookSchema)
async def get_book(id: int, db: Session = Depends(get_db)):
    return db.query(models.Book).options(joinedload(models.Book.authors)).\
         where(models.Book.id == id).one()


@app.get("/books", response_model=List[schemas.BookSchema])
 async def get_books(db: Session = Depends(get_db)):
     return db.query(models.Book).options(joinedload(models.Book.authors)).all()


@app.get("/authors/{id}", response_model=schemas. AuthorSchema)
async def get_author(id: int, db: Session = Depends(get_db)):
    return db.query(models.Author).options(joinedload(models.Author.books)).\
        where(models.Author.id == id).one()


@app.get("/authors", response_model=List[schemas.AuthorSchema])
async def get_authors(db: Session = Depends(get_db)):
    return db.query(models.Author).options(joinedload(models.Author.books)).all()



from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, Table
from sqlalchemy.orm import relationship

from .database import Base



book_authors = Table('book_authors', Base.metadata,
    Column('book_id', ForeignKey('books.id'), primary_key=True),
    Column('author_id', ForeignKey('authors.id'), primary_key=True)
)

class Book(Base):
    __tablename__ = 'books'
    id = Column(Integer, primary_key=True)
    title = Column(String, nullable=False)
    authors = relationship("Author", secondary="book_authors", back_populates='books', uselist=False)

class Author(Base):
    __tablename__ = 'authors'
    id = Column(Integer, primary_key=True)
    name = Column(String, nullable=False)
    books = relationship("Book", secondary="book_authors", back_populates='authors')




class BookAuthors(BaseModel):
    book_id:int
    author_id: int

    class Config:
        orm_mode = True       

class Book(BaseModel):
    title:str
    author: str

    class Config:
        orm_mode = True       


class Author(BaseModel):
    name:str
    books:str

    class Config:
        orm_mode = True       


class AuthorBase(BaseModel):
    id: int
    name: str

    class Config:
        orm_mode = True

class BookBase(BaseModel):
    id: int
    title: str

    class Config:
        orm_mode = True

class BookSchema(BookBase):
    authors: List[AuthorBase]

class AuthorSchema(AuthorBase):
    books: List[BookBase]
```

### Description

I am trying to make a post operation to the author and book endpoint, but upon execution to store i into the database. i keep getting this error

WARNING:  WatchFiles detected changes in 'app/main.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [50005]
INFO:     Started server process [50631]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
WARNING:  WatchFiles detected changes in 'app/main.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [50631]
INFO:     Started server process [50660]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:58608 - "GET /docs HTTP/1.1" 200 OK
INFO:     127.0.0.1:58608 - "GET /openapi.json HTTP/1.1" 200 OK
INFO:     127.0.0.1:58610 - "GET /authors HTTP/1.1" 200 OK
WARNING:  WatchFiles detected changes in 'app/main.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [50660]
INFO:     Started server process [50826]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
WARNING:  WatchFiles detected changes in 'app/main.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [50826]
INFO:     Started server process [50853]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:58612 - "GET /docs HTTP/1.1" 200 OK
INFO:     127.0.0.1:58612 - "GET /openapi.json HTTP/1.1" 200 OK
INFO:     127.0.0.1:58614 - "POST /author HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py", line 401, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/applications.py", line 269, in __call__
    await super().__call__(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/exceptions.py", line 93, in __call__
    raise exc
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__
    await self.app(scope, receive, sender)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 670, in __call__
    await route.handle(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 266, in handle
    await self.app(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 65, in app
    response = await func(request)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/home/timmy/projects/apiCaller/./app/main.py", line 167, in create_author
    create_author = models.Author(**form.dict())
  File "<string>", line 4, in __init__
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/state.py", line 482, in _initialize_instance
    manager.dispatch.init_failure(self, args, kwargs)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/state.py", line 479, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/decl_base.py", line 1166, in _declarative_constructor
    setattr(self, k, kwargs[k])
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/attributes.py", line 459, in __set__
    self.impl.set(
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/attributes.py", line 1563, in set
    raise TypeError(
TypeError: Incompatible collection type: str is not list-like
^CINFO:     Shutting down
^CINFO:     Finished server process [50853]
ERROR:    Traceback (most recent call last):
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 638, in lifespan
    await receive()
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/uvicorn/lifespan/on.py", line 137, in receive
    return await self.receive_queue.get()
  File "/usr/lib/python3.8/asyncio/queues.py", line 163, in get
    await getter
asyncio.exceptions.CancelledError

INFO:     Stopping reloader process [44704]
(env) timmy@timmy-HP-ProBook-4420s:~/projects/apiCaller$ uvicorn app.main:app --reload
INFO:     Will watch for changes in these directories: ['/home/timmy/projects/apiCaller']
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [53003] using WatchFiles
INFO:     Started server process [53005]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:58634 - "GET /docs HTTP/1.1" 200 OK
INFO:     127.0.0.1:58634 - "GET /openapi.json HTTP/1.1" 200 OK
INFO:     127.0.0.1:58634 - "POST /author HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py", line 401, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/applications.py", line 269, in __call__
    await super().__call__(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/exceptions.py", line 93, in __call__
    raise exc
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__
    await self.app(scope, receive, sender)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 670, in __call__
    await route.handle(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 266, in handle
    await self.app(scope, receive, send)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/starlette/routing.py", line 65, in app
    response = await func(request)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/home/timmy/projects/apiCaller/./app/main.py", line 167, in create_author
    create_author = models.Author(**form.dict())
  File "<string>", line 4, in __init__
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/state.py", line 482, in _initialize_instance
    manager.dispatch.init_failure(self, args, kwargs)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/state.py", line 479, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/decl_base.py", line 1166, in _declarative_constructor
    setattr(self, k, kwargs[k])
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/attributes.py", line 459, in __set__
    self.impl.set(
  File "/home/timmy/projects/apiCaller/env/lib/python3.8/site-packages/sqlalchemy/orm/attributes.py", line 1563, in set
    raise TypeError(
TypeError: Incompatible collection type: str is not list-like

### Operating System

Linux

### Operating System Details

Ubuntu 20.04

### FastAPI Version

Fastapi Version 0.78.0

### Python Version

python verson 3.8.10

**Comment by JarroVGIT** [comment]
Please provide the endpoints that you are using. You gave 4 GET endpoints in your FastAPI app, but the error is on POST endpoint 'author'. Without that information, I would say that the `uselist=False` seems suspicious given the error that you are getting, but that is no more than speculation on my part.

**Comment by Timmy-Edibo** [comment]
> Please provide the endpoints that you are using. You gave 4 GET endpoints in your FastAPI app, but the error is on POST endpoint 'author'. Without that information, I would say that the `uselist=False` seems suspicious given the error that you are getting, but that is no more than speculation on my part.

Thanks for reaching out JarroVGT
The get endpoints works properly. but when i try to use any of the post endpoint, the error keep coming up.

The Author and Books endpoint for creating post to be precise.

I also removed the "uselist=False" but still didn't work
Thanks once again..

**Comment by JarroVGIT** [comment]
I understand that your GET works fine, but you didn't actually provided us with your POST endpoints. We only see these:

```python
@app.get("/books/{id}", response_model=schemas.BookSchema)
async def get_book(id: int, db: Session = Depends(get_db)):
    return db.query(models.Book).options(joinedload(models.Book.authors)).\
         where(models.Book.id == id).one()


@app.get("/books", response_model=List[schemas.BookSchema])
 async def get_books(db: Session = Depends(get_db)):
     return db.query(models.Book).options(joinedload(models.Book.authors)).all()


@app.get("/authors/{id}", response_model=schemas. AuthorSchema)
async def get_author(id: int, db: Session = Depends(get_db)):
    return db.query(models.Author).options(joinedload(models.Author.books)).\
        where(models.Author.id == id).one()


@app.get("/authors", response_model=List[schemas.AuthorSchema])
async def get_authors(db: Session = Depends(get_db)):
    return db.query(models.Author).options(joinedload(models.Author.books)).all()
```

None of these are the POST endpoints that is causing the error. Without it, it is very hard to help.

**Comment by Timmy-Edibo** [comment]
> I understand that your GET works fine, but you didn't actually provided us with your POST endpoints. We only see these:
> 
> ```python
> @app.get("/books/{id}", response_model=schemas.BookSchema)
> async def get_book(id: int, db: Session = Depends(get_db)):
>     return db.query(models.Book).options(joinedload(models.Book.authors)).\
>          where(models.Book.id == id).one()
> 
> 
> @app.get("/books", response_model=List[schemas.BookSchema])
>  async def get_books(db: Session = Depends(get_db)):
>      return db.query(models.Book).options(joinedload(models.Book.authors)).all()
> 
> 
> @app.get("/authors/{id}", response_model=schemas. AuthorSchema)
> async def get_author(id: int, db: Session = Depends(get_db)):
>     return db.query(models.Author).options(joinedload(models.Author.books)).\
>         where(models.Author.id == id).one()
> 
> 
> @app.get("/authors", response_model=List[schemas.AuthorSchema])
> async def get_authors(db: Session = Depends(get_db)):
>     return db.query(models.Author).options(joinedload(models.Author.books)).all()
> ```
> 
> None of these are the POST endpoints that is causing the error. Without it, it is very hard to help.

Oh sorry...I'm just going to add it now...Thought i did..

@app.post('/author/')
async def create_author(form: schemas.Author, db: Session = Depends(get_db)):
    create_author = models.Author(**form.dict())

    db.add(create_author)
    db.commit()
    db.refresh(create_author)
    return create_author
    
    
@app.post('/books')
async def create_book(form: schemas.Book, db: Session = Depends(get_db)):
    create_book = models.Book(**form.dict())

    db.add(create_book)
    db.commit()
    db.refresh(create_book)
    return create_book
    
    Thanks.. I just took note of that

**Comment by JarroVGIT** [comment]
Alright, we are almost there; now what does your post look like? How do you post it? (e.g. via the automated docs? Or via cURL?) Please provide as many details as possible. The issue might be input related.

**Comment by Timmy-Edibo** [comment]
> Alright, we are almost there; now what does your post look like? How do you post it? (e.g. via the automated docs? Or via cURL?) Please provide as many details as possible. The issue might be input related.

ok..I'm attaching screenshots of the endpoints
![Screenshot 2022-07-02 at 10-20-40 FastAPI - Swagger UI](https://user-images.githubusercontent.com/83462106/176994605-94607fce-cc4f-4f2b-9ef5-65f8899e3d3b.png)
![Screenshot 2022-07-02 at 10-16-58 FastAPI - Swagger UI](https://user-images.githubusercontent.com/83462106/176994607-435b15df-f290-4624-ae16-4b84d359c1cf.png)
![Screenshot 2022-07-02 at 10-16-25 FastAPI - Swagger UI](https://user-images.githubusercontent.com/83462106/176994609-b17cfc97-9b0f-4620-b33b-72ac40e2f3fc.png)
![Screenshot 2022-07-02 at 10-15-55 FastAPI - Swagger UI](https://user-images.githubusercontent.com/83462106/176994612-7ac0c43b-1ef4-47f0-bebf-055224a2f060.png)

**Comment by Timmy-Edibo** [comment]
@JarroVGIT 
I think i have found where the issue is coming from. but i'll need help getting around it.

In both endpoints for Author and books,
"name": "string",
"author": "string"

"name": "string",
"books": "string"

the error is coming from the author and books filed, i guess i don't know what kind of inputs its expecting..
Kindly help review the model, to know what kind of input the fields are to recieve..

thanks..

**Comment by JarroVGIT** [comment]
The issue is that you provide a string in your request body rather than a list of Book objects. I am on my phone right now so can’t provide an example request body, but there is your problem.

**Comment by Timmy-Edibo** [comment]
> The issue is that you provide a string in your request body rather than a list of Book objects. I am on my phone right now so can’t provide an example request body, but there is your problem.

Thank you so much @JarroVGIT 
I'm very grateful.

You can in you spare time just help with the solution. regards

**Comment by JarroVGIT** [comment]
Alright, you haven't provided the full code so I am unable to make a working example. But I can give you some pointers. For starters:
```python
class Author(BaseModel):
    name:str
    #books:str EDIT: this is wrong replace by the following:
    books: List[Book] 

    class Config:
        orm_mode = True 
```
The property `books` already implies multiple (which is correct, one Author can have multiple Books). But you set it as a `str`, rather than a `List[Books]`.  Your request body should look like this:

```json
{
    "name": "author name",
    "books": [
        {
            "title": "fastapi book"
        }
    ]
}
```    
I don't know what you have classified as `models` and what are `schemas`. I would expect that the Pydantic models are `schemas` (such as `Author(BaseModel)`) and the table definitions are the `models`. You named them the same (both have an `Author` object) which is highly confusing. Also, and again this is now a recurring topic in this issue, you have provided an incomplete code base for us to really help you out.

```python
async def create_author(form: schemas.Author, db: Session = Depends(get_db)):
    create_author = models.Author(**form.dict())
```
What is the definition of `schemas.Author` and what is the definition of `models.Author`? 

Lastly; please don't quote full responses in your replies, it makes it incredibly unreadable. If you really need to quote, then be very precise and only quote that part that you want to reply on.

**Comment by Timmy-Edibo** [comment]
> 

Here are the definitions..

schemas.Author

class Author(BaseModel):
    name:str
    books:str

    class Config:
        orm_mode = True       

model.Author

class Author(Base):
    __tablename__ = 'authors'
    id = Column(Integer, primary_key=True)
    name = Column(String, nullable=False)
    books = relationship("Book", secondary="book_authors", back_populates='authors')

And thanks for the piece of advice..really do appreciate

**Comment by JarroVGIT** [comment]
Have you tried changing your `Author(BaseModel)` as I suggested above? Because the error is that there is a mismatch between your Pydantic model (which expects a string) and your database model (which expects it to be a list of books). 

If that doesn't work, I'll have a look tomorrow. If you want to help, please create a minimal reproducible example and put it in a public repository and provide the link. That way, I don't have to recreate your codebase from bits and pieces all over in this thread. Thanks in advance.

**Comment by Timmy-Edibo** [comment]
> Have you tried changing your `Author(BaseModel)` as I suggested above? Because the error is that there is a mismatch between your Pydantic model (which expects a string) and your database model (which expects it to be a list of books).
> 
> If that doesn't work, I'll have a look tomorrow. If you want to help, please create a minimal reproducible example and put it in a public repository and provide the link. That way, I don't have to recreate your codebase from bits and pieces all over in this thread. Thanks in advance.

yeah..just tried it...
Here's what i got

![Screenshot 2022-07-02 at 13-31-42 FastAPI - Swagger UI](https://user-images.githubusercontent.com/83462106/177000915-131d55f4-d282-4bcc-b0f3-b4cad05c14bb.png)

**Comment by Timmy-Edibo** [comment]
> > Have you tried changing your `Author(BaseModel)` as I suggested above? Because the error is that there is a mismatch between your Pydantic model (which expects a string) and your database model (which expects it to be a list of books).
> > If that doesn't work, I'll have a look tomorrow. If you want to help, please create a minimal reproducible example and put it in a public repository and provide the link. That way, I don't have to recreate your codebase from bits and pieces all over in this thread. Thanks in advance.
> 
> yeah..just tried it... Here's what i got
> 
> ![Screenshot 2022-07-02 at 13-31-42 FastAPI - Swagger UI](https://user-images.githubusercontent.com/83462106/177000915-131d55f4-d282-4bcc-b0f3-b4cad05c14bb.png)

Kindly find attached, the github link to the projects.thanks
https://github.com/Timmy-Edibo/apiCaller
